demos/can-component/accordion.html

```diff
@@ -14,23 +17,14 @@
 			<li>JavaScriptMVC</li>
 		</ul>
 	</my-panel>
-</my-accordion>
-</script>
-
-
-<script src="../../node_modules/steal/steal.js" dev-bundle main="@empty">
-var Component = require("can-component");
-var stache = require("can-stache");
-var DefineMap = require("can-define/map/map");
-var DefineList = require("can-define/list/list");
-var canViewModel = require("can-view-model");
+</my-accordion>`;
 
-var Panel = DefineMap.extend({
+const Panel = DefineMap.extend({
 	active: "boolean",
 	title: "string"
 });
 
-var AccordionViewModel = DefineMap.extend({
+const AccordionViewModel = DefineMap.extend({
```

Comment:
We should probably change these eventually to the "standard" component syntax, without everything broken out.